mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -506,6 +507,16 @@ public void listRoles(UserManagementProto.ListRolesRequest request, StreamObserv
         responseObserver.onCompleted();
     }
 
+    @Override
+    public void getActorWorkloadCredentials(com.cloudera.thunderhead.service.usermanagement.UserManagementProto.GetActorWorkloadCredentialsRequest request,
+                                            io.grpc.stub.StreamObserver<com.cloudera.thunderhead.service.usermanagement
+                                                .UserManagementProto.GetActorWorkloadCredentialsResponse> responseObserver) {
+
+        GetActorWorkloadCredentialsResponse.Builder respBuilder = GetActorWorkloadCredentialsResponse.getDefaultInstance().toBuilder();
```

Comment:
I think we need to have some value returned here. Otherwise  folks won't be able to log in to clusters created from a locally running FMS and CBD.
We'll also need to advertise the password that this response would correspond to to engineering.

Reply:
Added hardcoded impl back and added Docs to mention string value of hash.